vt-digital-libraries-platform/resolution-service
# pytest setup

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2529) (:star:)

# What does this Pull Request do? (:star:)
- Setup pytest
- Add test cases
- Check if noid is exist then perform update
- remove file and packages that no longer needed

# How should this be tested?
- See the update readme.md and follow the instruction in the `Tests` section

# Interested parties
Tag (@whunter ) interested parties

(:star:) Required fields
